.travis.yml

```diff
@@ -1,40 +1,49 @@
-    on:
-      distributions: sdist bdist_wheel
-      repo: azavea/grout
-      branch: develop
+
+python: '3.6'
+
+services:
+  - docker
+
+jobs:
+  include:
+    - stage: test
+      script: ./scripts/test app py27
+      install: skip
+      name: "Python 2.7"
+    - script: ./scripts/test app py34
+      install: skip
+      name: "Python 3.4"
+    - script: ./scripts/test app py35
+      install: skip
+      name: "Python 3.5"
+    - script: ./scripts/test app py36
+      install: skip
+      name: "Python 3.6"
+    - script: ./scripts/test app py37
+      install: skip
+      name: "Python 3.7"
+    - stage: deploy
+      script: skip
+      deploy:
+        - provider: pypi
+          user: azavea
+          skip_existing: true
+          password:
+            secure: gVivn0yuENFhivdr7ZbE6WQc8SNa9ocWph2UWdfhPLXxpzGivjTJmHNHyU+8VyXqD25rxLDm6KYFOgxKKzdjBsNvle9Jw4DvJkVnO06RJiYJK8SYAE5HElowRpBHu1UNTjh86U1YI6StRsvJGB8zogAshHhhQmpGsmNICkuQc1VCJJHtTOYVLsIVJodAAc98BfR2u+4guz6h3MVjWgIUhk+kqVMv9oq87ryPlWxu5A2w3ZP+abLFdZ2cXLNsx8ZIzmAEYgCOcJkX17Q4mTNWX7kfOy7hYT3TlQhAK90i5qvIlkZmGy6NCuLV3ToxEyGvY4tTqRs4WvWgf3+4P2DblCVbMwednnNunS1cu64FD8Jll3KGV83vWvnJRCF4LgI80E/WN+1T6EbQU71NLb+JDyZsT0UiCFAMoGu+jdg4iqKP3Y5GyOd7ozlhXfoCl+uNHmOaO/idtt2FVSL3+vSMbu1HWsXST2sX1ojOM9OVRoiebxPVzCUDYyc/Z1OMHfp/S8xbGFhjhPwFXTBJ3xzUzCyWUme5oeJhKJuQPXBPXr2yyaUyVIobmBaJnhXPI/1N666tLihYYpNzgs7l5ZNllA0By3KQ9kZjN/WcOlELjnOVlUUNOI9b9b0FvjvFjKrKt2No4VXihRN0VeamZolpO02Kt+mnvcPRpVSKKUqEtGA=
+          on:
+            distributions: sdist bdist_wheel
+            repo: azavea/grout
+            branch: master
+        - provider: pypi
+          user: azavea
+          skip_existing: true
+          server: https://test.pypi.org/legacy/
+          password:
+            secure: gVivn0yuENFhivdr7ZbE6WQc8SNa9ocWph2UWdfhPLXxpzGivjTJmHNHyU+8VyXqD25rxLDm6KYFOgxKKzdjBsNvle9Jw4DvJkVnO06RJiYJK8SYAE5HElowRpBHu1UNTjh86U1YI6StRsvJGB8zogAshHhhQmpGsmNICkuQc1VCJJHtTOYVLsIVJodAAc98BfR2u+4guz6h3MVjWgIUhk+kqVMv9oq87ryPlWxu5A2w3ZP+abLFdZ2cXLNsx8ZIzmAEYgCOcJkX17Q4mTNWX7kfOy7hYT3TlQhAK90i5qvIlkZmGy6NCuLV3ToxEyGvY4tTqRs4WvWgf3+4P2DblCVbMwednnNunS1cu64FD8Jll3KGV83vWvnJRCF4LgI80E/WN+1T6EbQU71NLb+JDyZsT0UiCFAMoGu+jdg4iqKP3Y5GyOd7ozlhXfoCl+uNHmOaO/idtt2FVSL3+vSMbu1HWsXST2sX1ojOM9OVRoiebxPVzCUDYyc/Z1OMHfp/S8xbGFhjhPwFXTBJ3xzUzCyWUme5oeJhKJuQPXBPXr2yyaUyVIobmBaJnhXPI/1N666tLihYYpNzgs7l5ZNllA0By3KQ9kZjN/WcOlELjnOVlUUNOI9b9b0FvjvFjKrKt2No4VXihRN0VeamZolpO02Kt+mnvcPRpVSKKUqEtGA=
+          on:
+            distributions: sdist bdist_wheel
+            repo: azavea/grout
+            branch: develop
```

Comment:
Does this deploy on merges to both `develop` and `master`?

We may want to limit it to merges to `master`, for stability, and to maintain tagged releases for versioning.
(I understand this was an existing configuration setting, but something to consider.)

Reply:
We deploy the package to the testing instance of PyPi (https://test.pypi.org) from `develop`, and the production instance (https://pypi.org) from `master`. [Derek suggested this pattern](https://github.com/azavea/ashlar/pull/81#pullrequestreview-133227669) back when we were first setting up deployments for the project. I think it still makes sense to use the test instance for checking deployments, but I do think that pushing from `develop` maybe doesn't make much sense from a git-flow perspective, since version bumps aren't supposed to happen on `develop` anyway (by my understanding). Maybe a better pattern would be to instruct Travis to deploy to the test instance from `release-*` branches?